tests/test_validation.py

```diff
@@ -14,6 +14,12 @@
                                                     '..', 'example_listings'))
 
 
+def drop_api_exported_fields_so_that_api_import_will_validate(data):
```

Comment:
Would it be better to fix the example listing rather than re-introduce this method here?  The example listing should be up-to-date with what the API will handle.